storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -14,7 +14,7 @@ Familiarity with xref:../../storage/understanding-persistent-storage.adoc#unders
 
 To create CSI-provisioned persistent volumes (PVs) that mount to vSphere storage assets, {product-title} installs the vSphere CSI Driver Operator and the vSphere CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
 
-* *vSphere CSI Driver Operator*: The Operator provides a storage class, called `thin-csi`, that you can use to create persistent volumes claims (PVCs). The vSphere CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
+* *vSphere CSI Driver Operator*: The Operator provides a storage class, called `thin-csi`, that you can use to create persistent volumes claims (PVCs). The vSphere CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage. You can disable this default storage class if desired (see xref:../../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-sc-disable_persistent-storage-csi-vsphere[Disabling the default storage class]).
```

Comment:
This is true for other CSI drivers, are we going to update these as well?

Reply:
@gcharot The way I read the epic, it seemed like vSphere was in scope for 4.13, with the intent to extend it to all CSI drivers in subsequent releases. Not true?

Reply:
True at first, this is what drove the initial use case (use case 1) but we had requests from other customers on other CSI drivers (use case 2 & 3). Also we wanted to have it consistent across CSI drivers and avoid any special behavior with vsphere. Check the epic's affected components.
The list of drivers are listed as issue links of https://issues.redhat.com/browse/STOR-947 - this is pretty much the list of drivers we deploy via CSO
@dobsonj could you please confirm that list is exhaustive? thx

Reply:
The list in STOR-947 is accurate, as of 4.13 we plan to support the `storageClassState` field in the ClusterCSIDriver for the following operators (i.e. any operator that already creates a CSI StorageClass):
- alibaba-disk-csi-driver-operator
- aws-ebs-csi-driver-operator
- azure-disk-csi-driver-operator
- azure-file-csi-driver-operator
- gcp-pd-csi-driver-operator
- ibm-vpc-block-csi-driver-operator
- openstack-cinder-csi-driver-operator
- ovirt-csi-driver-operator
- vmware-vsphere-csi-driver-operator

Reply:
Fixed.